.github/ISSUE_TEMPLATE/xtable_bug_report.yml

```diff
@@ -0,0 +1,68 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+---
+name: XTable bug report 🐞
+description: Problems, bugs and issues with Apache XTable
+labels: ["bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you very much for submitting feedback to XTable to help XTable develop better.
```

Comment:
Suggestion on wording: `Thank you very much for submitting feedback to Apache XTable (Incubating) to help make the project better.`

Reply:
> Suggestion on wording: `Thank you very much for submitting feedback to Apache XTable (Incubating) to help make the project better.`

Updated.